package.json

```
@@ -17,6 +17,7 @@
     "dev": "turbo run dev",
     "dev:not-apps": "turbo run dev --filter=@lssm/lib* --filter=@lssm/module*",
     "build": "turbo run build --filter=!@lssm/lib-service.ui-kit",
```

Biome

[error] The key build was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)